addons/project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -37,17 +37,22 @@
             </ProjectRightSidePanelSection>
             <ProjectRightSidePanelSection
                 name="'profitability'"
-                show="showProjectProfitability"
+                show="!!this.state.data.profitability_items"
             >
                 <t t-set-slot="title" owl="1">
                     Profitability
                 </t>
-                <ProjectProfitability
-                    data="state.data.profitability_items"
-                    labels="state.data.profitability_labels"
-                    formatMonetary="formatMonetary.bind(this)"
-                    onClick="(params) => this.onProjectActionClick(params)"
-                />
+                <span t-if="showProjectProfitability" class="text-muted fst-italic">
```

Comment:
But why don't you keep the same result for `showProjectProfitability`, and just swap the content of the spans? :monocle_face:

Reply:
Hello, @auon-odoo applied changes. Thanks :slightly_smiling_face: